addons/ai/functions/fnc_waypointParadrop.sqf

```diff
@@ -0,0 +1,63 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Scripted waypoint that makes a group paradrop at the waypoint's position.
+ *
+ * Arguments:
+ * 0: Group <GROUP>
+ * 1: Waypoint Position <ARRAY>
+ *
+ * Return Value:
+ * Waypoint Finished <BOOL>
+ *
+ * Example:
+ * [group, [0, 0, 0]] call zen_ai_fnc_waypointParadrop
+ *
+ * Public: No
+ */
+
+#define MOVE_DELAY 3
+
+params ["_group", "_waypointPosition"];
+
+private _waypoint = [_group, currentWaypoint _group];
+_waypoint setWaypointDescription localize LSTRING(Paradrop);
+
+private _vehicle = vehicle leader _group;
+
+// Exit if the vehicle is not an aircraft
+if !(_vehicle isKindOf "Air") exitWith {true};
+
+private _driver = driver _vehicle;
+private _skill = skill _driver;
+
+// Increase the skill of the pilot for better flying
+_driver setSkill 1;
+_driver allowFleeing 0;
+
+private _nextMove = CBA_missionTime;
+
+waitUntil {
```

Comment:
Per ACE3 coding guidelines it is disallowed to use it.

Reply:
Scripted waypoints have to be scheduled.